src/Nowruz/modules/general/components/iconDropDown/index.tsx

```diff
@@ -26,7 +26,12 @@ export const IconDropDown: React.FC<IconDropDownProps> = (props) => {
 
   return (
     <div className="flex flex-col items-end relative">
-      <IconButton className={`${css.avatarBtn} ${open && `${css.avatarBtnOpen}`}`} disableRipple onClick={handleClick}>
+      <IconButton
+        className={`${css.avatarBtn} ${open && `${css.avatarBtnOpen}`}`}
+        disableRipple
+        onClick={handleClick}
+        aria-label="icon-button"
```

Comment:
same here

Reply:
@marjan-ara For this one it doesn't matter because it was just an aria-label not referencing any ID, it doesn't need to be unique as we are only telling that this button will have this label.